src/app/components/experiment-timeline/experiment-timeline.component.ts

```diff
@@ -50,6 +50,16 @@ export class ExperimentTimelineComponent implements OnInit, OnDestroy {
         { value: 1, viewValue: "Only steps with uncleared substeps" },
         { value: -1, viewValue: "Only steps with cleared substeps" }
     ];
+    resultQuality: ExperimentResultQuality | "" = "";
+    resultQualityValues: SelectValue[] = [
```

Comment:
It would be great (but not required) if the SelectValue interface is generic in the type for `value` so that the value entries here could be properly type checked.

Reply:
The use of a `SelectValue` interface is now avoided completely for the `resultQualityValues`.